daemon/lokinet.cpp

```diff
@@ -11,6 +11,10 @@
 #include <dbghelp.h>
 #endif
 
+#ifdef __APPLE__
+#include <llarp/util/logging/apple_logger.hpp>
+#endif
+
```

Comment:
We don't build this at all on apple anymore, should drop these changes (this and the one below).